fili-core/src/main/java/com/yahoo/bard/webservice/metadata/SegmentMetadataLoader.java

```diff
@@ -156,7 +156,6 @@ public void invoke(JsonNode rootNode) {
                 if (segmentMetadata.isEmpty()) {
                     LOG.warn("Empty segment metadata detected when loading table '{}'", table.getFactTableName());
                 }
-                table.resetColumns(segmentMetadata, dimensionDictionary);
```

Comment:
I will remove it since it was already `Deprecated` and are using the new `DataSourceMetadataLoader` now.